addons/udes_stock/views/stock_location.xml

```diff
@@ -25,6 +25,8 @@
                     <field name="u_date_last_checked_correct"
                            attrs="{'readonly': [('u_date_last_checked_correct', '!=', False)]}"/>
                     <field name="u_quant_policy" readonly="1"/>
+                    <field name="u_location_is_countable"/>
```

Comment:
I don't think we need to worry about that, only users with permissions to edit locations would be able to change this anyway and they can also do other actions such as making locations inactive.

In practice it is most likely this would never be changed, but I think it's good to give the stock manager or whoever else has permission the option just in case.